### PR TITLE
Update Contributors list for release v0.1.38

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- As we are approaching release date (02.03.2018) of version 0.1.38, update list of contributors.
- Thank you:
  - @OnceUponALoop 
  - @ptitoliv 
  - @hooksie1 
  - @nzwulfin 
  - @rlucente-se-jboss 

#### Rationale:

- Proposing this to check with new contributors name and email to use in list.

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ The following people have contributed to the SCAP Security Guide project
...
 * Gabe Alford <redhatrises@gmail.com>
+* Firas AlShafei <firas.alshafei@us.abb.com>
 * Christopher Anderson <cba@fedoraproject.org>
 * Chuck Atkins <chuck.atkins@kitware.com>
 * Ryan Ballanger <root@rballang-admin-2.fastenal.com>
 * Molly Jo Bault <Molly.Jo.Bault@ballardtech.com>
 * Jose Luis BG <bgjoseluis@gmail.com>
 * Joseph Bisch <joseph.bisch@gmail.com>
 * Jeffrey Blank <blank@eclipse.ncsc.mil>
-* Olivier Bonhomme <ptitoliv@ptitoliv.net>
```
Comment (mpreisler) [inline Contributors.md]: ?

Comment (yuumasato) [reply Contributors.md]: He is added later, but just as Olivier.
His name might have changed in later commits.

Comment (mpreisler) [reply Contributors.md]: Full name is better than just first name, what about creating an alias.

Comment (yuumasato) [reply Contributors.md]: Good idea, name mapped.

**Pull request merged**